tee-worker/service/src/enclave/api.rs

```diff
@@ -52,10 +52,9 @@ pub fn enclave_init(config: &Config) -> EnclaveResult<Enclave> {
 	if use_token {
 		match File::open(&token_file) {
 			Err(_) => {
-				info!(
-					"[-] Token file {} not found! Will create one.",
-					token_file.as_path().to_str().unwrap()
-				);
+				#[allow(clippy::unwrap_used)]
+				let file_str = token_file.as_path().to_str().unwrap();
```

Comment:
Maybe sth like `token_file.as_path().to_string_lossy()` would avoid the need to `unwrap` altogether?